css-canvas-medical/index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row underline prl-7perc row-1-ptb">
+      <div class="col-3">
+        <div class="row ht-100">
+          <img class="blue-logo" src="images/logo-blue.svg" alt="blue logo">
+        </div>
+      </div>
+      <div class="col-3-two pl-row1">
+        <div class="row ht-100 font-avenir font-17px">
+          <div class="col-flex">
+            <a class="txt-lightgray" href="#">Product</a>
+          </div>
+          <div class="col-flex">
+            <a class="txt-lightgray" href="#">Implementation</a>
+          </div>
+          <div class="col-flex">
+            <a class="txt-lightgray" href="#">About Us</a>
+          </div>
+          <div class="col-flex">
+            <a class="txt-lightgray" href="#">Blog</a>
+          </div>
+          <div class="col-flex mrl-0">
+            <a class="txt-white back-blue contact-button" href="#">Contact Us</a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row prl-7perc mt-70">
+      <div class="back-hero row">
```

Comment:
You shouldn't have a `row` as a child of a `row`. You likely want this `row` to be a child of a `column-full` class element